modules/dr-restoring-cluster-state.adoc

```diff
@@ -186,6 +186,186 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
+. If you have lost the majority of your control plane hosts, leading to etcd quorum loss, delete and recreate the control plane machines (also known as the master machines), one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
++
+If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps. Otherwise, you must create the new master using the same method that was used to originally create it.
+
+.. Obtain the machine for the unhealthy member.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api -o wide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                        PHASE     TYPE        REGION      ZONE         AGE     NODE                           PROVIDERID                              STATE
+clustername-8qw5l-master-0                  Running   m4.xlarge   us-east-1   us-east-1a   3h37m   ip-10-0-131-183.ec2.internal   aws:///us-east-1a/i-0ec2782f8287dfb7e   stopped <1>
```

Comment:
I know it's probably going to be a pain, but these example node names, etc. match the examples from the unhealthy member doc. This restore doc uses some different names in the examples. You'll have to coordinate and update these examples so that they match the examples used throughout the rest of the procedure.

Let me know if you need help with what needs to change.

Worker node examples in the outputs should be fine to stay as is, since I think this is the only time that they're shown in the procedure